README.md

```diff
@@ -69,6 +69,12 @@ Deploy it on the server:
 $ make deploy
 ```
 
+## Updating libdds Submodule
+
+Submodule updates can be simple done by checking out any commit hash which means
+branches, tags and plane commit hash numbers. Next step is `git add libdds`
+which stages the submodule update like other changes.
+
```

Comment:
I have no idea what this means. Please help me out! Also. this could frighten potential newcomers. It might be better to document it otherwise.

Reply:
I was able to deploy successfully using this branch, so all looks good once the doc is updated.

Reply:
> I have no idea what this means. Please help me out! Also. this could frighten potential newcomers. It might be better to document it elsewhere.

Good point. I moved the documentation to CONTRIBUTING.md and tried to make it easier to understand.
Did I manage to write it in a manner which can be understood?